### PR TITLE
Adding simple migration example

### PR DESCRIPTION
**Pull request by ptorru**

### CODE REVIEW

File: simple_migration/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+2. Redefine `OPENAI_BASE_URL` to point to `https://text.octoai.run/v1`
+3. Change the model name to an Open Source model, for example: `llama-2-13b-chat-fp16`
+
+
+## What you will use
+We will only use Python and the OpenAI's Python SDK.
+
+## Instructions
+
+- Set up python virtual environment
```
Comment (chailandau) [inline simple_migration/README.md:22]: Maybe add a link to a tutorial / resource on python's site?

File: simple_migration/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+```bash
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+- Install the pip requirements in your local python virtual environment
```
Comment (chailandau) [inline simple_migration/README.md:29]: Does pip get put into code? `pip`

File: simple_migration/README.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+To run this example app, there are two simple steps to take:
+
+- Get an OctoAI API token by following [these instructions](https://octo.ai/docs/getting-started/how-to-create-octoai-api-token/).
+- Expose the token in a new `OCTOAI_TOKEN` environment variable:
+
+```bash
+export OCTOAI_TOKEN=<your-token>
+```
+- Now source the provided `environment.sh` file:
```
Comment (chailandau) [inline simple_migration/README.md:45]:
```suggestion
- Source the provided `environment.sh` file:
```

File: simple_migration/README.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+```bash
+source environment.sh
+```
+
+If you prefer, you can also directly paste your token in the file.
```
Comment (chailandau) [inline simple_migration/README.md:50]:
```suggestion
If you prefer, you can paste your token directly into the file.
```

Comment (chailandau) [reply simple_migration/README.md:50]: Is this a security risk? I do not know backend stuff so could be fine

File: simple_migration/README.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+You will notice that we have done some of the changes for you:
```
Comment (chailandau) [inline simple_migration/README.md:52]:
```suggestion
We have completed some of these tasks for you:
```

File: simple_migration/README.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+* We have already added the `OPENAI_API_BASE` URL for you.
+* We have already changed line 5 from the example to select Llama-2-13B as the model to use.
```
Comment (chailandau) [inline simple_migration/README.md:54]:
```suggestion
* We have already changed line 5 from the example to select the `Llama-2-13B` model.
```

File: simple_migration/README.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+### Running the application
+
+Run `response.py` script to chat with the LLM hosted endpoint.
```
Comment (chailandau) [inline simple_migration/README.md:58]:
```suggestion
Run the `response.py` script to chat with the hosted LLM endpoint.
```

Comment (chailandau) [reply simple_migration/README.md:58]: I could be misunderstanding your intent here, so feel free to ignore

File: simple_migration/README.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+```bash
+python3 response.py
+```
+
+### Example output
```
Comment (chailandau) [inline simple_migration/README.md:63]: This seems like it should have content but it is empty right now

File: simple_migration/README.md (hunk 4 above)
Comment (ptorru) [reply simple_migration/README.md:50]: This is a simple demo, I think it is safe.

**Review by chailandau — APPROVED** [review]

File: simple_migration/README.md (hunk 2 above)
Comment (ptorru) [reply simple_migration/README.md:29]: I hesitated, some people seem to use PIP, but after checking their docs they stick to pip, no `code` formatting. I will follow that :-)

**Review by tmoreau89 — APPROVED** [review]
LGTM

**Pull request merged**